profile/src/v100/address/non_fungible_global_id.rs

```diff
@@ -0,0 +1,238 @@
+use crate::prelude::*;
+
+use radix_engine_common::address::AddressBech32Decoder;
+use radix_engine_toolkit_json::models::scrypto::non_fungible_global_id::{
+    SerializableNonFungibleGlobalId as EngineSerializableNonFungibleGlobalId,
+    SerializableNonFungibleGlobalIdInternal as EngineSerializableNonFungibleGlobalIdInternal,
+};
+
+use transaction::prelude::NonFungibleGlobalId as EngineNonFungibleGlobalId;
+
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    SerializeDisplay,
+    DeserializeFromStr,
+    derive_more::Display,
+    uniffi::Record,
+)]
+#[display("{}", self.to_canonical_string())]
+pub struct NonFungibleGlobalId {
```

Comment:
Would be nice if we use just wrap `transaction::prelude::NonFungibleGlobalId` in a newtype and have our own JSON serialization and deserialization and UniFFI export of it...